speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/SteerDecision.cpp

```diff
@@ -1,10 +1,18 @@
 #include "SteerDecision.h"
-
-void SteerDecision::GetIndicateCommands()
+#include "Mediator.h"
+void SteerDecision::RunIndicateCommands()
 {
-
+	if(m_steerAmount < -0.3f)
+	{
+		SMediator::GetInstance().CarController.ShowUI(UI_TYPE_TURN_LEFT);
+		return;
+	}
+	if(m_steerAmount > 0.3f)
+	{
+		SMediator::GetInstance().CarController.ShowUI(UI_TYPE_TURN_RIGHT);
+	}
```

Comment:
Where do the values -0.3 and 0.3 come from?

Reply:
it just needed a value and can be changed later